package.json

```diff
@@ -3,7 +3,13 @@
   "version": "0.3.0",
   "main": "index.js",
   "license": "Apache-2.0",
-  "files": ["index.d.ts"],
+  "files": [
+    "index.d.ts"
+  ],
+  "dependencies": {
+    "@types/formidable": "*",
+    "@types/node": "*"
```

Comment:
We can avoid `@types/formidable` but not `@types/node`. Maybe we should mark `@types/node` as peer dependency and add it to `devDependencies` so that we still can reference it in our test.